libsolidity/interface/OptimiserSettings.h

```diff
@@ -41,7 +41,7 @@ struct OptimiserSettings
 
 			// should have good "compilability" property here.
 
-			"eul"                      // Run functional expression inliner
+			"Tpeul"                    // Run functional expression inliner
```

Comment:
Added LiteralRematerialiser (T) before UnusedFunctionParamterPruner (p) because of https://github.com/ethereum/solidity/issues/9653